.github/workflows/build.yml

```diff
@@ -60,6 +60,7 @@ jobs:
 
     - name: Publish logs
       uses: actions/upload-artifact@v1
+      if: ${{ always() }}
```

Comment:
Interesting. What does that mean for the actions that execute after a step decorated with `if: ${{ always() }}`? Do they execute even if the first step - but not the previous one - failed?

Reply:
It just applies to the specific step you decorate. The default if you don't specify anything is that it calls `success()`: [docs](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions)

Reply:
Very nice! Thanks 👏